src/Sylius/Component/Translation/Model/AbstractTranslatable.php

```diff
@@ -19,15 +19,11 @@
 abstract class AbstractTranslatable implements TranslatableInterface
 {
     /**
-     * Translations.
-     *
      * @var TranslationInterface[]
```

Comment:
Let's stick to current convention to avoid inconsistency and if we decide to decouple we will do it everywhere at once.